deployment/deploy-az-demo-bootstrapper.ps1

```diff
@@ -0,0 +1,37 @@
+# ------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation.  All rights reserved.
+#  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+# ------------------------------------------------------------
+Param(
+    [string]
+    [Parameter(mandatory=$true)]
+    $ApplicationName
+)
+
+# ----- Copy scripts from source location
+$baseLocation = "https://raw.githubusercontent.com/suneetnangia/distributed-az-edge-framework/feature/docs-update"
+
+Invoke-WebRequest -Uri "$baseLocation/deployment/deploy-core-infrastructure.ps1" -OutFile "deploy-core-infrastructure.ps1"
+Invoke-WebRequest -Uri "$baseLocation/deployment/deploy-core-platform.ps1" -OutFile "deploy-core-platform.ps1"
+Invoke-WebRequest -Uri "$baseLocation/deployment/deploy-app.ps1" -OutFile "deploy-app.ps1"
+
+mkdir -p bicep/modules
+
+Invoke-WebRequest -Uri "$baseLocation/deployment/bicep/core-infrastructure.bicep" -OutFile "./bicep/core-infrastructure.bicep"
+Invoke-WebRequest -Uri "$baseLocation/deployment/bicep/app.bicep" -OutFile "./bicep/app.bicep"
+Invoke-WebRequest -Uri "$baseLocation/deployment/bicep/modules/aks.bicep" -OutFile "./bicep/modules/aks.bicep"
+Invoke-WebRequest -Uri "$baseLocation/deployment/bicep/modules/azurestorage.bicep" -OutFile "./bicep/modules/azurestorage.bicep"
+Invoke-WebRequest -Uri "$baseLocation/deployment/bicep/modules/eventhub.bicep" -OutFile "./bicep/modules/eventhub.bicep"
+
+./deploy-core-infrastructure.ps1 -ApplicationName $ApplicationName
+
+clear
+./deploy-core-platform.ps1 -ApplicationName $ApplicationName
+
+clear
+./deploy-app.ps1 -ApplicationName $ApplicationName -AKSClusterResourceGroupName $env:RESOURCEGROUPNAME -AKSClusterName $env:AKSCLUSTERNAME
+
+clear
+Write-Host "-------------------------------------------------------------------------------------------------------------------------------------------" -ForegroundColor Yellow
```

Comment:
Maybe you should use Write-Title here

Reply:
Yeah, I think if we are considering to use Write-Title in multiple script files, we should may be look at moving that function to a common/shared module or something?

Reply:
Will take a look at having a shared module for common functions in PoSh.